go/tasks/pluginmachinery/ioutils/remote_file_output_writer.go

```diff
@@ -71,10 +72,11 @@ func (w RemoteFileOutputWriter) Put(ctx context.Context, reader io.OutputReader)
 	return fmt.Errorf("no data found to write")
 }
 
-func NewRemoteFileOutputPaths(_ context.Context, store storage.ReferenceConstructor, outputPrefix storage.DataReference) RemoteFileOutputPaths {
```

Comment:
given that now there's a new datareference as part of `io.RawOutputPaths` would it help to add some more functions to `RemoteFileOutputReader`?

Reply:
Nope, we never want to read that data. Now that we call RawOutput that should be more clear.